### ISSUE TITLE
Questions about learning rate decay

### ISSUE THREAD

**Issue by Jiaolong** [thuml/CDAN#20]
In the paper section 4.1, it is said:
ηp = η0(1 + αp)−β, where p is the training progress changing from 0 to 1

I understand p = iter_num/max_iters

However, in the following pytorch code: 
https://github.com/thuml/CDAN/blob/f7889063b76fca0b9a7147c88103d356531924bd/pytorch/lr_schedule.py#L3
and also in the tensorflow code:
https://github.com/thuml/CDAN/blob/master/tensorflow/train.py#L13

`p = iter_num` is used instead of `p = iter_num/max_iters`

Can you explain about this? Thanks!
